docs/src/validator/validator_length_rule.md

```diff
@@ -0,0 +1,20 @@
+## Length Rule
+The lengths of properties in string format provided by `cobra.Command` can be controlled using the Length Rule.
+
+## How to use
+1. Create a length rule by providing the limits for each attribute you want to control
+```go
```

Comment:
Could you do some default validator that does initialize those values and allow users to override somethjng if needed.